CTFd/api/v1/challenges.py

```diff
@@ -753,7 +753,7 @@ def get(self, challenge_id):
         ).all()
 
         for f in challenge_files:
-            response.append({"id": f.id, "type": f.type, "location": f.location})
+            response.append({"id": f.id, "type": f.type, "location": f.location, "sha1sum": f.sha1sum})
```

Comment:
Yes this makes sense thanks for adding this. It should align with the responses for `/api/v1/files`.